src/spatial_ref/srs.rs

```diff
@@ -562,6 +562,8 @@ impl SpatialRef {
     /// * `Ok(None)` - if node or attribute are not found (C library will return `nullptr`) or attribute contains no value,
     /// * `Err(_)` - if there is a string conversion error.
     ///
+    /// **Note:** this fuction panics when child > [libc::c_int::MAX].
```

Comment:
Can you format it [like this](https://doc.rust-lang.org/src/core/cell.rs.html#791-793)? Also, should it have some backquotes (`child > `[`libc::c_int::MAX`])? I'm not sure how the links work here.

Reply:
I made a correction.